samples/app-sso/csharp/app_sso/TeamsAppManifest/manifest.json

```diff
@@ -0,0 +1,136 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.7",
+  "version": "1.0.0",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.microsoft.teams.samples",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://example.azurewebsites.net",
+    "privacyUrl": "https://example.azurewebsites.net/privacy",
+    "termsOfUseUrl": "https://example.azurewebsites.net/termsofuse"
+  },
+  "icons": {
+    "color": "icon-color.png",
+    "outline": "icon-outline.png"
+  },
+  "name": {
+    "short": "BotTabMeSSO",
+    "full": "Teams Single Sign On Sample"
+  },
+  "description": {
+    "short": "Bot Tab ME SSO Scenario",
```

Comment:
Update description to `This sample app demonstrates SSO for Tab Bot and ME (Link Unfurl, Action and Search Config)`

Reply:
Description is  updated on Manifest